contracts/BlastNativeYield.sol

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.20;
+
+import {IBlast, YieldMode as IBlast__YieldMode, GasMode as IBlast__GasMode} from "./interfaces/IBlast.sol";
+import {BlastPoints} from "./BlastPoints.sol";
+/**
```

Comment:
add an empty line between line 6 and 7

Reply:
https://github.com/LooksRare/contracts-libs/pull/103/commits/f80299b624bb29f3c05c21cf2ed9fbe11b714574